### ISSUE TITLE
Coding a simple neural network from scratch

### ISSUE THREAD

**Issue by Pravanop** [WebClub-NITK/Hacktoberfest-2k20#60]
### Description
Write functions for a forward pass and backward propagation for a single artificial neuron with sigmoid activation. Do not use TensorFlow, PyTorch, Keras, Caffe or any other such libraries.
### Details
- Technical Specifications: Preferably code in Python, use the NumPy library.
- Type of issue: Single
- Time Limit: 1 day after claiming the issue.

### Issue requirements / progress
- [x] Write functions for forward and backward propagation for a single neuron.

### Resources
https://numpy.org/

### Directory Structure
Make a folder called simple_neuron under the Machine learning folder and add a single file with your code.

#### Note
1. Please claim the issue first by commenting here before starting to work on it.
2. Once you are done with the task and have created a Pull Request, please tag @ to request a review.

**Comment by buttercrab** [comment]
Can I do this task?

**Comment by Pravanop** [comment]
> Can I do this task?

Sure, go ahead!

**Comment by krithikvaidya** [comment]
@Pravanop closing the issue since the PR has been merged.
